src/deflate.c

```diff
@@ -2,6 +2,9 @@
 #include "libdeflate.h"
 #include <Python.h>
 
+// Use the stable ABI
+#define Py_LIMITED_API 0x030b00f0
```

Comment:
This should be protected on Python version. You only want to define it for 3.11+.

Reply:
(Actually, the easiest way is to do the define from CMake)

Reply:
But how will I know the Python version before including `<Python.h>`? Defining it after the include didn't actually remove the symbols.

Reply:
Getting 0.7 out is perfectly fine! I'm trying this out in a PR.